src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -199,6 +199,7 @@ void Analyzer::Display() {
       }
 
       ShowGain("r-squared", &m_rSquared);
+      ShowGain("Sim RMSE", m_plot.GetRMSE());
```

Comment:
Can we add a tooltip explaining what this is? I don't expect everyone to be familiar with this acronym.

Reply:
Let me know if the tooltip is descriptive enough